cilium-cli/connectivity/check/context.go

```diff
@@ -303,6 +303,21 @@ func (ct *ConnectivityTest) SetupAndValidate(ctx context.Context, extra SetupHoo
 	if err := ctx.Err(); err != nil {
 		return err
 	}
+
+	setupAndValidate := ct.setupAndValidate
+	if ct.Params().Perf {
+		setupAndValidate = ct.setupAndValidatePerf
```

Comment:
nice, we are getting one step closer to finally separating it from connectivity tests :sweat_smile: